### ISSUE TITLE
TooltipTrigger should be keyboard accessible

### ISSUE THREAD

**Issue by macfarlandian** [Recidiviz/web-libraries#88]
**What needs to be done? Why does it need to be done?**
Users should be able to focus a tooltip trigger with the tab key to reveal the tooltip.

**Additional context**
n/a
